mobile-v3/lib/main.dart

```diff
@@ -164,11 +160,14 @@ class _DeciderState extends State<Decider> {
           children: [
             FutureBuilder<String?>(
               future: HiveRepository.getData('token', HiveBoxNames.authBox),
```

Comment:
Thanks @Karagwa , just also seen a small enhancement request somewhere which can be scheduled for future PRs. It might not be urgent. Please see attached screenshot.
![Uploading Screenshot 2025-02-15 at 10.35.15.png…]()